workspaces/default/themes/light-theme/assets/styles/site.css

```diff
@@ -446,7 +456,7 @@ a.tag-link.is-active {
   padding: 3rem;
   align-items: start;
   background: rgba(255,255,255);
-  box-shadow: 0 1px 2px rgba(0,0,0,0.1), 0 0 10px rgba(0,0,0,0.08);
+  box-shadow: 0 1px 2px var(--border, grey);
```

Comment:
Wondering if this should stay as a drop shadow, cc @jitachi

Reply:
Let's say yes. 🤔

Reply:
Apologies, realized that my comment wasn't very clear.

Right now it's a solid drop-shadow that's very heavy handed, perhaps it should stay as an reduced alpha channel of black.